_posts/2018-07-13-KafkaSchemaRegistryPart1.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+title: Kafka - Confluent Schema Registry. Part 1
+categories: Kafka
+tags: [Kafka, Schema registry, Apache Avro, Confluent]
+author: NikitaZhevnitskiy
+---
+
+# Introduction
+
+Let's talk about Kafka Schema Registry essentials.
+
+# Repository
+
+Repository contains examples of schema evolution - full compatibility.
+[Kafka-schema-registry-poc](https://github.com/sysco-middleware/kafka-schema-registry-poc)
+
+## Content: 
+
+1. What is schema registry and why we need it?  
```

Comment:
Maybe make sense to clarify that is "Confluent" Schema Registry (there are other schema registries out there)